cmd/siac/rentercmd.go

```diff
@@ -1,5 +1,23 @@
 package main
 
+// TODO: If you run siac from a non-existant directory, the abs() function does
```

Comment:
I think those are pretty good `programming tasks`. Would be good if you created issues for them too.